_posts/2016-05-07-sensitivity-analysis.md

```diff
@@ -1,6 +1,85 @@
 ---
 layout: post
 title: Sensitivity Analysis
-img: image-5.png
+img: sensitivity.png
 ---
 
+Sensitivity analysis helps to the study of how the variations in the outputs of a numerical or 
+physical system can be assigned to different sources of variation in its inputs.
```

Comment:
What is a numerical physical system? I have never heard the phrase.

Reply:
I mean either a numerical system, like a simulator, or a physical one, like the amount of CO2 produced by a car.

Reply:
Maybe worth rephrasing then, I agree with Maren that this phrase looks weird. Especially "numerical system" part

Reply:
done